### PR TITLE
Added achievements page

### PR DESCRIPTION
**Pull request by agundy**
This adds an easy interface for admins to add historic events to the site. Displays as a single page timeline currently.

### CODE REVIEW

**Comment by seveibar** [comment]
code is fine- gotta pull it down and do a run through

**Comment by seveibar** [comment]
looks like it broke the test suite

**Comment by agundy** [comment]
Removed what should be the offending test

**Comment by agundy** [comment]
the page doesn't refresh but the list refreshes and Year/Month/Day is pretty standard, what would you rather it be?

File: client/app/achievements/achievements.controller.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      ach.$delete(function() {
+        $scope.achievements = Achievement.query();
+      });
+    };
+
+    $scope.submit = function() {
+      Achievement.save($scope.achievement, function(){
+          $scope.achievement = {};
+        });
+      $scope.achievements = Achievement.query();
```
Comment (agundy) [inline client/app/achievements/achievements.controller.js]: moved this. up a line

**Comment by seveibar** [comment]
I think the US standard is month day year- ill call it a :whale: or whatever you say-

**Comment by TheoBr** [comment]
This looks good but (visually) looks iffy on the home page. Perhaps the "Achievements" hotlink should be moved to the menu bar instead of existing in a separate space?

**Comment by agundy** [comment]
Going to bump this  discussion to a new issue if you don't mind @TheoBr ?

**Comment by agundy** [comment]
@TheoBr please comment on #496 looks like I can't tag you since you aren't a part of the repo and you haven't commented on the issue yet

**Pull request merged**